src/opencl.cpp

```diff
@@ -0,0 +1,242 @@
+/*
+ * Adapted from an example found here https://stackoverflow.com/questions/4979504/fast-rgb-yuv-conversion-in-opencl
+ * Copyright 2019 Scott Moreau
+ *
+ */
+
+#include <iostream>
+
+#include "opencl.hpp"
+
+
+static char const *cl_source_str = "									\n\
```

Comment:
Instead of adding all of these `\n`, use C++11 raw strings.

Reply:
I couldn't figure out how to do this. The call to clCreateProgramWithSource() expects a const char**.

Reply:
I got it now, done.